component/general/materialComponents/Card1.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
```

Comment:
What's with the name of this `Card1`? What does it mean?

Reply:
We have 2 different material-ui components to render each hnn rule.

- Expansion panels (the one implemented)
- Cards 

Card1 and Card2 just allow you to change the material component without having to modify code. We should probably agree on whether to use Card or Expansion Pane and delete the other one. I did both because I was not sure which one would fit better.

Expansion panel is more straight forward, and the expansion/contraction feature is simpler.

Reply:
I removed Card2 and renamed Card1 to Card